blog/2023-10-17-tmai-september-23/index.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
Few of the metadata in this frontmatter are not necessary.
Read the [first steps](https://opensource.tcetmumbai.in/blog/blog-posting-guidelines#first-steps) of the posting guidelines for which metadata information should be used.